README.md

```diff
@@ -124,6 +124,39 @@ const mySteps: TourStep[] = [{
 
 You can also find a complete example [here](example/).
 
+## Built-in Helper Components
+
+You can take advantage of the already built-in customizable components. For example, our [TourBox] (https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/#tourbox) component can be used as a tooltip container for each step.
+
+
+```tsx
+const tourSteps: TourStep[] = [{
```

Comment:
Let's add all the imports from `@stackbuilders/react-native-spotlight-tour` to give better visibility of were things come from 🙂